engine/class_modules/sc_death_knight.cpp

```diff
@@ -4535,7 +4535,14 @@ struct death_coil_t : public death_knight_spell_t
     if ( p() -> buffs.sudden_doom -> check() )
       return 0;
 
-    return death_knight_spell_t::cost();
+    double cost = death_knight_spell_t::cost();
+
+    if ( p() -> legendary.deadliest_coil.ok() )
+    {
+      cost += (p() -> legendary.deadliest_coil->effectN( 1 ).base_value() / 10);
```

Comment:
You can use ``.resource( RESOURCE_RUNIC_POWER )`` instead of ``.base_value() / 10`` here.